TestResultSummaryService/routes/getTestByVersionInfo.js

```diff
@@ -0,0 +1,63 @@
+const { TestResultsDB } = require('../Database');
+
+/**
+ * getTestByVersionInfo returns the latest test builds with matched version
+ * 
+ * @param {string} version required. java version regex.

```

Comment:
wondering if we should call this param "regex" instead of version, to differentiate it from jdkVersion and to make it clear what you can pass in?